torch/utils/data/datapipes/utils/decoder.py

```diff
@@ -182,8 +199,16 @@ def torch_video(key, data):
     if extension not in "mp4 ogv mjpeg avi mov h264 mpg webm wmv".split():
         return None
 
-    # add `type: ignore` to avoid mypy's warning on import missing
-    import torchvision.io  # type: ignore
+    try:
+        import torchvision.io
+    except ImportError as e:
+        try:
+            import pip  # type: ignore
+            pip.main(['install', '--user', 'torchvision'])
```

Comment:
Please DO NOT install things on users behalf, throw error with instruction how to do it

Reply:
It makes sense.